mu/mu_debug.py

```diff
@@ -16,5 +16,19 @@
         sys.path.append(path_to_add)
 
 
+def debug(filename=None, *args):
+    """
+    Create a debug runner in a new process.
+
+    This is what the Mu debugger will drive. Uses the filename and associated
+    args found in sys.argv.
+    """
+    if filename is None:
+        print("Debugger requires a Python script filename to run.")
```

Comment:
Probably :) I think this wasn't actually my code change. It shows up as a diff because I had to rename the `mu-debug.py` to `mu_debug.py` to allow it to be imported IIRC.